### ISSUE TITLE
Option to avoid regenerate during build, and add regenerate command

### ISSUE THREAD

**Issue by Grinnz** [miyagawa/Dist-Milla#49]
I am opening this for discussion, as I'm not sure how people feel about it regarding back compat. The [regenerate](https://metacpan.org/pod/Dist::Zilla::App::Command::regenerate) command by @kentfredric provides an alternative to munging metadata files on every build, which some consider intrusive, especially contributors who just want to test a build. Implementation would be straightforward: use CopyFilesFromRelease instead of CopyFilesFromBuild, order it in the correct place for AfterRelease, set the root ReadmeAnyFromPod phase to release, and then use Regenerate::AfterReleasers on those two plugins. `milla regenerate` should work automatically.

The question is mostly, is this operation desired, and would people want to change their workflow?

CC @tokuhirom and @skaji for coordination with Minilla.

**Comment by Grinnz** [comment]
I've employed this as an option for my Starter bundle, and mbtiny also works this way, for what it's worth.

**Comment by skaji** [comment]
> especially contributors who just want to test a build. 

Usually, contributors does not even need to install milla.

**Comment by Grinnz** [comment]
Agreed, but some contributors want to, or are making changes to the Milla process rather than the module.

**Comment by Grinnz** [comment]
I'd also note, my personal preference as the author is often to make changes and test things, and regenerate the meta files in a separate commit.
